src/Cairo.jl

```diff
@@ -335,6 +365,18 @@ end
 @compat show(io::IO, ::MIME"image/png", surface::CairoSurface) =
    write_to_png(surface, io)
 
+function read_from_png_stream{T<:IO}(stream::T)
```

Comment:
Does this really have to be a separately-named function? Why not just make it a method of `read_from_png`?